app/services/transfer_destroy_service.rb

```diff
@@ -4,6 +4,8 @@ def initialize(transfer_id:)
   end
 
   def call
+    raise StandardError.new "Transfer cannot be deleted" unless transfer.deletable?
```

Comment:
You don't need to explicitly call out `StandardError`:

`raise "Transfer cannot be deleted" unless transfer.deletable?`

But can you add more information here? "Transfer cannot be deleted since an audit was performed after it."